ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -939,6 +939,11 @@ Rectangle {
         target: Global.dragArea
```

Comment:
I've got an idea. We could improve by disabling the Connections if the control is not visible
```
enabled: control.visible
```

Reply:
This also resolves the issue and, since it guards better against future mistakes, I think it's better.